src/pages/Contacts.js

```diff
@@ -1,26 +0,0 @@
-const Contacts = () => {
-    return ( 
-          <main className="section">
```

Comment:
format this file: main should be closer to the left side and the next tag (child) should be with additional spaces